nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -150,6 +151,32 @@ private static void reportInvalidReturnTypeError(
             errorMessage, analysis.buildDescription(tree), state, null));
   }
 
+  private static void reportMismatchedTypeForTernaryOperator(
+      Tree tree,
+      Type expressionType,
+      Type rhsType,
+      VisitorState state,
+      NullAway analysis,
+      boolean isTruePartOfTheExpression) {
```

Comment:
Rather than passing this boolean parameter, we should pass either the true or false part sub-expression as the `tree` argument.  Then the error message can be something like `"conditional expression must have type " + expressionType + ", but sub-expression has type " + rhsType + ", which has mismatched nullability type parameters".  We can get rid of the boolean parameter then

Reply:
will update the error message